proversity_reports_script/report_backend/learning_tracker_report.py

```diff
@@ -0,0 +1,91 @@
+"""
+Report backend for completion report.
+"""
+import csv
+
+import boto3
+
+from proversity_reports_script.report_backend.base import AbstractBaseReportBackend
+
+
+class LearningTrackerReportBackend(AbstractBaseReportBackend):
+    """
+    Backend for Completion report.
+    """
+
+    def __init__(self, *args, **kwargs):
+        extra_data = kwargs.get('extra_data', {})
+        super(LearningTrackerReportBackend, self).__init__(extra_data.get('SPREADSHEET_DATA', {}))
```

Comment:
Are you using the SPREADSHEET_DATA?, otherwise, we can avoid calling the super. @diegomillan

Reply:
Yes, But I prefer to delegate on configs for future support @Squirrel18